shell/common/rasterizer.cc

```diff
@@ -176,74 +189,71 @@ GrDirectContext* Rasterizer::GetGrContext() {
   return surface_ ? surface_->GetContext() : nullptr;
 }
 
-flutter::LayerTree* Rasterizer::GetLastLayerTree() {
-  return last_layer_tree_.get();
+flutter::LayerTree* Rasterizer::GetLastLayerTree(int64_t view_id) {
+  auto found = last_successful_tasks_.find(view_id);
+  if (found == last_successful_tasks_.end()) {
+    return nullptr;
+  }
+  return found->second.layer_tree.get();
 }
 
 void Rasterizer::DrawLastLayerTree(
```

Comment:
Should we update this method's name?

Reply:
I've changed it to `DrawLastLayerTrees`.